### ISSUE TITLE
Add by Library of Congress Number 

### ISSUE THREAD

**Issue by spaulaus** [eleybourn/Book-Catalogue#563]
Is it possible to implement adding a book by their Library of Congress Number. This would be extremely useful for adding books published before 1970.

**Comment by emdalton** [comment]
I agree, this would be extremely helpful. I have many older books with LoC numbers but no ISBN.
